### PR TITLE
Add declaration types for ie11

### PR DESCRIPTION
**Pull request by Luchanso**
Hi, can we fix typings for IE 11? 🤔

image

Was tested in my project - all correctly work, without breaking changes.
You can test it too `@luchanso/react-hook-form@6.8.0-rc.luchanso.1`

### CODE REVIEW

**Comment by bluebill1049** [comment]
Thanks, @Luchanso, looks fine by me. @kotarella1110 any objection?

File: src/index.ie11.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export * from './useForm';
+export * from './useFieldArray';
+export * from './useWatch';
+export * from './controller';
+export * from './useFormContext';
+export * from './logic';
+export * from './utils';
+export * from './types';
```
Comment (kotarella1110) [inline src/index.ie11.ts:8]: You can improve this as follows:

```suggestion
export * from '.';
```

Comment (kotarella1110) [inline src/index.ie11.ts:8]: You can improve this as follows:

```suggestion
export * from '.';
```

**Review by kotarella1110 — COMMENTED** [review]
Great! Thanks for your PR 💚

**Comment by bluebill1049** [comment]
Thanks for your review @kotarella1110 👍

**Pull request merged**